du/src/du.yml

```diff
@@ -0,0 +1,133 @@
+name: du
+version: '0.0.0'
+author: 'Julian Kaindl <kaindl745@gmail.com>'
+about: 'Summarize disk usage for the set FILEs, recursively for directories'
+args:
+    - FILE:
+          help: One or more file names
+          required: false
+          multiple: true
+    - line-end-null:
+          long: 'null'
+          short: '0'
+          help: Ends each output line with NUL, not newline
+    - all:
+          long: all
+          short: a
+          help: Prints counts for all files, not just directories
+    - apparent-size:
```

Comment:
All itens should have long and short options, if in the original doesn't have one, fell free to create

Reply:
Looks like on FreeBSD that one flag is `-A`